specs/Deployment/DeploymentAPI.md

```diff
@@ -221,6 +266,26 @@ namespace Microsoft.Windows.ApplicationModel.WindowsAppRuntime
         /// WindowsAppSDK main and singleton packages will be shut down forcibly if they are
         /// currently in use, when registering the WinAppSDK packages.
         Boolean ForceDeployment;
+
+        /// If not successful call DebugBreak()
+        [contract(DeploymentContract, 3)]
+        Boolean OnError_DebugBreak;
+
+        /// If not successful call DebugBreak() if a debugger is attached to the process
+        [contract(DeploymentContract, 3)]
+        Boolean OnError_DebugBreak_IfDebuggerAttached;
+
+        /// If not successful perform a fail-fast
+        [contract(DeploymentContract, 3)]
+        Boolean OnError_FailFast;
+
+        /// If not successful show UI
+        [contract(DeploymentContract, 3)]
+        Boolean OnError_ShowUI;
+
+        /// Do nothing (do not error) if the process lacks package identity
+        [contract(DeploymentContract, 3)]
+        Boolean OnNoPackageIdentity_NOOP;
```

Comment:
'Empty' doesn't quite convey it. Lingo lately is "packaged app" = has package identity and "unpackaged app" = lacks package identity.

Expressing it code-wise `if (not process.HasPackageIdentity) { ignore/skip/return }`. Coming up with a good phrase...

* OnNoPackageIdentity
* OnLacksPackageIdentity
* OnUnpackagedProcess

Nothing really struck me as all that wonderful. Any hot flashes of Eureka! by you?

Reply:
AllowUnpackagedDeployment ?   Getting the right lingo for Packaged/Identity is important but that isn't what feels unnatural to me.  This is an "InitializeOption", so I think we can express it in terms of what the developer is signing up for, instead of an implicit event in the initialize code.  ("OnNoPackageIdentity", On* sounds like an 'event' occurring but it's actually an initial condition of the deployment, right?)

Reply:
Why is this even an option here? The app developer shouldn't have to set this _and_ build an MSI, they just ... build the MSI.  What's this bit even trying to convey or avoid?

Reply:
RECOMMEND: Drop NoOp option and make behavior "DoNothingIfUnpackaged" i.e. NoOp option is always TRUE